app/src/main/java/ch/epfl/sdp/drone3d/service/impl/mission/ParallelogramMappingMissionService.kt

```diff
@@ -9,13 +9,15 @@ import ch.epfl.sdp.drone3d.model.mission.*
 import ch.epfl.sdp.drone3d.service.api.drone.DroneService
 import ch.epfl.sdp.drone3d.service.api.mission.MappingMissionService
 import com.mapbox.mapboxsdk.geometry.LatLng
-import java.lang.IllegalArgumentException
 import javax.inject.Inject
+import kotlin.math.PI
 
 class ParallelogramMappingMissionService @Inject constructor(val droneService: DroneService): MappingMissionService {
 
     companion object{
-        private const val cameraAngle = 0.0 // Suppose the drone is looking down
+        // The drone looks forward with a 60 degree angle compared to vertical
+        // which corresponds to a 30 degree angle compared to vertical
```

Comment:
I think there is a typo here, otherwise it's not clear what you meant

Reply:
Indeed, I meant horizontal